server/region_handler.go

```diff
@@ -536,12 +536,12 @@ func (t *regionHandlerTool) getMvccByRecordID(tableID, recordID int64) (*kvrpcpb
 	}
 
 	tikvReq := &tikvrpc.Request{
-		Type:     tikvrpc.CmdMvccGetByKey,
-		Priority: kvrpcpb.CommandPri_Normal,
+		Type: tikvrpc.CmdMvccGetByKey,
 		MvccGetByKey: &kvrpcpb.MvccGetByKeyRequest{
 			Key: encodeKey,
 		},
 	}
+	tikvReq.Context.Priority = kvrpcpb.CommandPri_Normal
```

Comment:
It seems  default priority  is `Normal`?

Reply:
Yes, this is a tiny refactor, the old code set it to `Normal` ...